g4g.py

```diff
@@ -1,19 +1,28 @@
 #!/usr/bin/python
 
 import requests
-from bs4 import BeautifulSoup
+
 from os import system
 from sys import exit
 from time import sleep
 from requests.exceptions import ConnectionError
+
+from bs4 import BeautifulSoup
+
 from article import Article
 
+
 BASE_URL = 'http://www.geeksforgeeks.org/'
 articles = []
 
-choice_to_category = {1: 'c', 2: 'c-plus-plus', 3: 'java', 4: 'python',
-                      5: 'fundamentals-of-algorithms',
-                      6: 'data-structures'}
+CHOICE_TO_CATEGORY_MAPPING = {
+    1: 'c',
+    2: 'c-plus-plus',
+    3: 'java',
+    4: 'python',
+    5: 'fundamentals-of-algorithms',
+    6: 'data-structures'
+}
```

Comment:
@CuriousLearner  I have use a code Linter to lint the code and at this line it was saying that The Closing Brackets does not match any Visual Indentation. That means the closing brackets of the code are not indented properly to match the starting brackets